docs/policies/security_incident_management_policy.md

```diff
@@ -0,0 +1,243 @@
+
+The analysis must include the following information:
+
+* identification of type of incident
+* assessment of impact
+* assessment of the scale of the incident
+* identification of classification or type of incident
+
+New services in scope for SbD implementation must manage security incidents ‘in house’ and portfolios and services who choose to manage security incidents ‘in house’ must report any incident through the IT Help Centre immediately on incident identification. This is so the SIM team are aware and have oversight of the incident in the event escalation or further support is required.
+Incidents managed ‘in house’ must not be above a Priority 4 (refer to Appendix 1, Criticality Matrix) without agreement and oversight from the SIM team as they may require some co-ordination to ensure an effective resolution.
+DfE must have policies and processes in place for reporting, managing, and resolving any information security incidents. All actions and decisions made during the response to incidents must be recorded.
+
+## Responsible Accountable Consulted Informed (RACI) matrix
+
+Where there is more than one role in a box this indicates the responsibility and whether the person or team is consulted or informed may change based on the team managing the incident (SIM team or ‘in house’ by portfolios and services), the incident priority (whether it is a P1, P2, P3 or P4) and is subject to change based on the incident itself.
+All acronyms used in the table below can be found in Appendix B.
+
+| Role<br><br>Activity                | CISO  | Head of Security Operations  | SIM Team | SOC | SRO / SO |Portfolios and Services| Staff|Relevant teams (for example, networks)|P1 Escalation Teams*|  
+|-------------------------------------|-------|------------------------------|---------|------|----------|-----------------------|------|--------------------------------------|--------------------|
+| Reporting a security incident| R /A |R| R / I|R|R|R|R|R||
+| Triage and manage P1 incidents| A | C | R |C||||C|C / I|
+| Triage and manage P2 / P3 incidents|A|C|R|C||||C||
+| Triage and manage P4 incidents|||R / C / I ||A|R||||
+| Escalation| A|C|R|R|C|R||||
+| Support remediation of P1 incidents| A|C|R|C|R|R|R|C|C|
+| Support remediation of P2 / P3 incidents|A|I|R|C / I|R||R|I|I|
+| Support remediation of P4 incidents|||R / C / I||A|R|R|||
+| Communicate incident updates| I|I|R|I|I|R / I|I|I|
+| Conduct post-incident reviews| A|C / I|R|C|C|R|C|C|C / I|
+|Policy and strategy| A/ R| C| I|I|I|I|I|I|I|
+|Awareness| I|I| R|I|I|R|I|C / I||
+ 
+ *P1 Escalation Teams include the National Cyber Security Centre (NCSC), the Gold Team, BAE Systems and Government Security Group (GSG) who may be consulted / informed during a P1 incident (see Appendix 1, Criticality Matrix) if necessary.
```

Comment:
We can use actual footnotes for this.

See: https://secure-by-design.security.education.gov.uk/example/#footnotes

```
[^1]: This footnote serves no real purpose other than to demonstrate how footnotes work.
      Click the arrow to go back to where you were.
```

Reply:
Good shout. I'll do another anchor too.